### PR TITLE
Update Coq-Elpi description

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Review by palmskog — CHANGES_REQUESTED** [review]
I think this description is way too generic. Please try to squeeze "λProlog" in there somewhere to differentiate from other (tactic) languages.

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Contributions welcome! Read the [contribution guidelines](CONTRIBUTING.md) first
 ### Plugins
 
 - [AAC Tactics](https://github.com/coq-community/aac-tactics) - Tactics for rewriting universally quantified equations, modulo associativity and commutativity of some operator.
-- [Coq-Elpi](https://github.com/LPCIC/coq-elpi) - Plugin for the Embeddable Lambda Prolog Interpreter.
+- [Coq-Elpi](https://github.com/LPCIC/coq-elpi) - Extension language to implement commands and tactics.
```
Comment (gares) [inline README.md:93]:
```suggestion
- [Coq-Elpi](https://github.com/LPCIC/coq-elpi) - Extension framework based on λProlog providing an extensive set of API to implement commands and tactics.
```

**Comment by gares** [comment]
better?

**Review by palmskog — APPROVED** [review]

**Comment by gares** [comment]
thanks

**Pull request merged**